public/Students/Leo Jayachandran/DynamicIndex/dynamicIndexMaker.js

```diff
@@ -253,7 +278,13 @@
 				let $termText = $(`<p>${alphabetisedIndex[i].terms[u].name}</p>`);
 				let $pagesText = $(`<div class = "pagesTextDiv"></div>`);
 				for (let j = 0; j < alphabetisedIndex[i].terms[u].pages.length; j++) {
-					$pagesText.append(`<a class = 'indexPages' title = "${alphabetisedIndex[i].terms[u].pages[j].pageName}" href = "${alphabetisedIndex[i].terms[u].pages[j].pageLink}">${alphabetisedIndex[i].terms[u].pages[j].pageName}</a> &#10; <br/>`);
+                    var pageToAdd = $(`<a></a>`).html(alphabetisedIndex[i].terms[u].pages[j].pageName).attr({
```

Comment:
If appropriate, use `let` or `const` instead of `var` so that your variables don't leak out.